dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/tooling/Instrumenter.java

```diff
@@ -354,49 +399,26 @@ public boolean isEnabled() {
     public boolean isApplicable(Set<TargetSystem> enabledSystems) {
       return false;
     }
+
+    protected final boolean isShortcutMatchingEnabled(boolean defaultToShortcut) {
+      return Config.get()
+          .isIntegrationShortcutMatchingEnabled(singletonList(name()), defaultToShortcut);
```

Comment:
Should this instead be:
```suggestion
          .isIntegrationShortcutMatchingEnabled(instrumentationNames, defaultToShortcut);
```

Reply:
Makes sense. Seems like the intent is to have the same behavior as the standard enable/disable so this should probably match that in my opinion. I'm ok with either way though

Reply:
I'm going to tackle this in a separate PR because I want to clean up how `Instrumenter` manages these names at the same time.